webpack.common.config.ts

```diff
@@ -643,9 +643,7 @@ module.exports = {
     new CleanWebpackPlugin({
       cleanAfterEveryBuildPatterns: ['**/*', '!*.html'],
     }),
-    new ForkTsCheckerWebpackPlugin({
-      checkSyntacticErrors: true
```

Comment:
Why was this removed?

Reply:
As an FYI, I actually looked into this when reviewing the PR. According to https://github.com/TypeStrong/fork-ts-checker-webpack-plugin#options it seems like this option is useful if we're using ts-loader in happyPackMode. Since we remove happyPackMode I guess that's why we're removing this.

But ... I'm not 100% sure, and a justification is still needed for why it is safe to do so. (One possible check could be if the PR that introduced this was also the one that introduced happypack.) So, deferring to @kevintab95 for comment.

Reply:
As per [this page](https://www.npmjs.com/package/fork-ts-checker-webpack-plugin), ts-loader does the syntax checking in transpileOnly mode. I double checked this by intentionally introducing a Syntax Error and seeing if the script reports it (and it did). Here's a screenshot.
![syntaxError](https://user-images.githubusercontent.com/11008603/75865285-5327aa80-5e29-11ea-9a83-8f3fc81b46d8.png)

Reply:
Here are screenshots reporting the build time taken by webpack before and after my changes.
Before changes:
![without_changes](https://user-images.githubusercontent.com/11008603/75867480-aea76780-5e2c-11ea-8da1-25e7e092e71f.png)

After Changes:
![with_changes](https://user-images.githubusercontent.com/11008603/75867520-bcf58380-5e2c-11ea-93c2-a8c5f0aa3910.png)

There seems to be a 10 sec delay since we are no longer using happyPack and threading.

Reply:
Thanks! I think, if this results in better stability, I'm willing to go with a 10-sec delay. @vojtechjelinek what do you think?

Reply:
Okay, that seems reasonable.

Reply:
Btw, my earlier comment was based on results from running the build on my local system. I tried comparing the build time on CircleCI and they look pretty much the same.

Before changes ([build](https://app.circleci.com/jobs/github/kevintab95/oppia/167)):
```
Using 1 worker with 2048MB memory limit
Hash: f61d7e9806e06cac49be
Version: webpack 4.41.6
Time: 25608ms
Built at: 03/04/2020 1:34:19 PM
```

After changes ([build](https://app.circleci.com/jobs/github/kevintab95/oppia/163)):
```
Using 1 worker with 2048MB memory limit
Hash: 28c8357ae57f3123f395
Version: webpack 4.41.6
Time: 25462ms
Built at: 03/04/2020 1:25:04 PM
```